Add Paul Armstrong's Foreman & AWX & FreeIPA blog

melcorr opened the pull request:
I aim to publish this on Monday morning! 

I left the dates as they are so it renders on preview.

@parmstro let me know what you think!

lzap commented on _posts/2021-09-03-automating-systems-with-foreman-awx-and-freeipa.md:
````diff
@@ -0,0 +1,193 @@
+                 snippet('ansible_tower_callback_service')) %>
+# Runs during first boot, removes itself
+systemctl enable ansible-callback
+```
````
Nitpick: There is a mechanism in systemd called `ConditionFirstBoot` and when set to True (https://www.freedesktop.org/software/systemd/man/systemd.unit.html) a service will be triggered only once. You could take advantage of this, however, not sure if this is available in EL7.

lzap replied:
I have just realized that this is coming from Foreman, is it?

Anyway, I did not mean this as a hard requirement or anything, it was just a sidenote. Feel free to keep it as is.

lzap commented on _posts/2021-09-03-automating-systems-with-foreman-awx-and-freeipa.md:
```diff
@@ -0,0 +1,193 @@
+<p float='left' align='left'>
+  <img src="/static/images/blog_images/2021-09-06-foreman-axw-freeipa-tutorial/0-foreman-awx-cockpit.png"  />
+</p>
+
```
This is the place where article would benefit from further explanation of how Foreman and AWX fit together. What the process is. Because frankly, it is pretty tough to understand how it all works.

I havent seen the images, chances are there is some diagram that explains it.

lzap commented:
Nice, thanks.

parmstro commented:
Hi Lukáš,

Thanks for the input, I will try to make the necessary changes today and
get the article updated.

Cheers,

Paul

On Fri, Sep 3, 2021 at 2:36 AM Lukáš Zapletal ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In _posts/2021-09-03-automating-systems-with-foreman-awx-and-freeipa.md
> <https://github.com/theforeman/theforeman.org/pull/1893#discussion_r701620770>
> :
>
> > +layout: post
> +title: Automating Systems with Foreman, AWX, and FreeIPA
> +date: 2021-09-02
> +author: Paul Armstrong
> +tags:
> +- foreman
> +---
> +
> +Foreman not only can ensure that our systems have a consistent view of our desired content, but also can provision those systems identically across many physical, virtual and public cloud environments. Whether we are using network or image based provisioning, there may be a limit to the configuration and integrations that we can apply to a system with kickstart or cloud-init during the build. Using Ansible in addition to kickstart opens up additional possibilities. In this post, we will look at how we integrate Foreman Provisioning with AWX to help us deliver consistent complete solutions repeatedly in any environment and across them!
> +
> +<!--more-->
> +
> +<p float='left' align='left'>
> +  <img src="/static/images/blog_images/2021-09-06-foreman-axw-freeipa-tutorial/0-foreman-awx-cockpit.png"  />
> +</p>
> +
>
> This is the place where article would benefit from further explanation of
> how Foreman and AWX fit together. What the process is. Because frankly, it
> is pretty tough to understand how it all works.
>
> I havent seen the images, chances are there is some diagram that explains
> it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/theforeman/theforeman.org/pull/1893#pullrequestreview-745750581>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMAI2IBOJ4RNEEPUR5LQBLUABUFXANCNFSM5DJGOOVA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Paul Armstrong
Principal Technical Specialist
Red Hat Technical Sales Organization
647-285-0044

lzap commented:
One thing I noticed is that on one screenshot job id is 11 (callback url dialog) but on the other (foreman parameters) it is 10. But we get the idea.

I saw your work in progress dialog, looks great, that will ensure readers are on the same page.

Great, looks good. Thanks.

melcorr commented:
Hey @parmstro 
Can you let me know what changes to make ? 
Or feel free to add a commit. Either works for me!

parmstro commented:
Hi Melanie,

I have 1 diagram with a caption to add . I will send it this morning, then
we are good to go.

Paul

On Thu, Sep 9, 2021 at 5:07 AM Melanie Corr ***@***.***>
wrote:

> Hey @parmstro <https://github.com/parmstro>
> Can you let me know what changes to make ?
> Or feel free to add a commit. Either works for me!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/theforeman/theforeman.org/pull/1893#issuecomment-915903396>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMAI2NEDLWE24PQCCNLYWTUBB2NRANCNFSM5DJGOOVA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Paul Armstrong
Principal Technical Specialist
Red Hat Technical Sales Organization
647-285-0044

melcorr committed:
Add Paul Armstrong's Foreman & AWX & FreeIPA blog

* update publication date
* remove one image
* address lzap feedback
* add new image with overview

melcorr commented:
I added in an overview diagram and some additional explanatory text from Paul. 
I think we have addressed everything. I am going to publish and advertise this on Monday morning. 
Thank you so so much @parmstro
